src/ServicePulse.Host/vue/src/composables/serviceFailedMessage.ts

```diff
@@ -23,10 +23,15 @@ export async function useRetryEditedMessage(
   id: string,
   editedMessage: Ref<{
     messageBody: string;
-    headers: unknown[];
+    headers: any[];
```

Comment:
Please, give this a type!

Reply:
The issue is that I have no idea how to do it 😆 any hint is welcome, I'm totally new to TypeScript

Reply:
{ key: string; value: string }[] should work... although that may require a typecast at callsites